src/System.Runtime.Extensions/tests/System/MathTests.netcoreapp.cs

```diff
@@ -305,5 +383,276 @@ public static void Clamp_MinGreaterThanMax_ThrowsArgumentException()
+        [InlineData( 2.1861299583286618,       1.1283791670955126,      CrossPlatformMachineEpsilon * 10)]  // expected:  (2 / sqrt(pi))
+        [InlineData( 2.6651441426902252,       1.4142135623730950,      CrossPlatformMachineEpsilon * 10)]  // expected:  (sqrt(2))
+        [InlineData( 2.7182818284590452,       1.4426950408889634,      CrossPlatformMachineEpsilon * 10)]  // expected:  (log2(e))
+        [InlineData( 2.9706864235520193,       1.5707963267948966,      CrossPlatformMachineEpsilon * 10)]  // expected:  (pi / 2)
+        [InlineData( 4.9334096679145963,       2.3025850929940457,      CrossPlatformMachineEpsilon * 10)]  // expected:  (ln(10))
+        [InlineData( 6.5808859910179210,       2.7182818284590452,      CrossPlatformMachineEpsilon * 10)]  // expected:  (e)
+        [InlineData( 8.8249778270762876,       3.1415926535897932,      CrossPlatformMachineEpsilon * 10)]  // expected:  (pi)
+        [InlineData( double.PositiveInfinity,  double.PositiveInfinity, 0.0)]
+        public static void Log2(double value, double expectedResult, double allowedVariance)
+        {
+            AssertEqual(expectedResult, Math.Log2(value), allowedVariance);
+        }
+
+        [Theory]
+        [InlineData(double.NegativeInfinity, double.PositiveInfinity, double.PositiveInfinity)]
+        [InlineData(double.MinValue, double.MaxValue, double.MaxValue)]
+        [InlineData(double.NaN, double.NaN, double.NaN)]
+        [InlineData(-0.0, 0.0, 0.0)]
+        [InlineData(2.0, -3.0, -3.0)]
+        [InlineData(3.0, -2.0, 3.0)]
+        [InlineData(double.PositiveInfinity, double.NaN, double.PositiveInfinity)]
+        public static void MaxMagnitude(double x, double y, double expectedResult)
+        {
+            AssertEqual(expectedResult, Math.MaxMagnitude(x, y), 0.0);
+        }
+        
+        [Theory]
+        [InlineData(double.NegativeInfinity, double.PositiveInfinity, double.NegativeInfinity)]
+        [InlineData(double.MinValue, double.MaxValue, double.MinValue)]
+        [InlineData(double.NaN, double.NaN, double.NaN)]
+        [InlineData(-0.0, 0.0, -0.0)]
+        [InlineData(2.0, -3.0, 2.0)]
+        [InlineData(3.0, -2.0, -2.0)]
+        [InlineData(double.PositiveInfinity, double.NaN, double.PositiveInfinity)]
+        public static void MinMagnitude(double x, double y, double expectedResult)
+        {
+            AssertEqual(expectedResult, Math.MinMagnitude(x, y), 0.0);
+        }
+
+        [Theory]
+        [InlineData( double.NegativeInfinity,  unchecked((int)(0x80000000)),  double.NegativeInfinity,  0)]
+        [InlineData( double.NaN,               unchecked((int)(0x7FFFFFFF)),  double.NaN,               0)]
+        [InlineData( 0,                        unchecked((int)(0x80000000)),  0,                        0)]
+        [InlineData( 0.11331473229676087,     -3,                             0.014164341537095108,     CrossPlatformMachineEpsilon / 10)]
```

Comment:
Do you want to add at least 1 negative `x` number that isn't negative infinity?

Reply:
Yes, I thought I already had, but apparently not 😄

Reply:
Fixed.